service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java

```diff
@@ -42,6 +42,7 @@ private static JudgeAndLegalAdvisor migrateJudgeAndLegalAdvisor(JudgeAndLegalAdv
             .otherTitle(allocatedJudge.getOtherTitle())
             .judgeLastName(allocatedJudge.getJudgeLastName())
             .judgeFullName(allocatedJudge.getJudgeFullName())
+            .judgeEmailAddress(allocatedJudge.getJudgeEmailAddress())
```

Comment:
When we migrate judge objects in create order, notice of proceedings etc the judge object associated with those events does not look like it captures judgeEmailAddress. It looks like this does not break anything (I see you've got a green build) but we might loose emailAddress in these cases. Is that a problem? Can you check wether this is the case?

Reply:
I will check thanks.

Reply:
As discussed, I will remove the migration of the judge's email address. As for create order, notice of proceedings etc, only a label for the allocated judge is retained. If there is a later requirement to use the judge's email address on another event other than Allocated Judge, it should be handle then.

Reply:
Done.